packages/babel-plugin-transform-runtime/src/index.ts

```diff
@@ -85,6 +91,14 @@ export default declare((api, options: Options, dirname) => {
 
   const corejsVersion = rawVersion ? Number(rawVersion) : false;
 
+  const {
+    moduleName = corejsVersion === 3
+      ? "@babel/runtime-corejs3"
+      : corejsVersion === 2
+      ? "@babel/runtime-corejs2"
+      : "@babel/runtime",
+  } = options;
+
```

Comment:
Can we just leave it where it was? After https://github.com/babel/babel/pull/15426/files#diff-6b555d8887bc858212b04534b4146442f8880fcf26df080baab49b90b1c10fb4R191-R192.

Reply:
I slightly prefer to have it at the top so that we have (roughly, because there is the logic to compute the `corejsVersion` variable):
- get the various options, with their default values
- validate all the options
- throw about old options
- the rest of the plugin logic

Reply:
Makes sense, can we move https://github.com/babel/babel/pull/15426/files#diff-6b555d8887bc858212b04534b4146442f8880fcf26df080baab49b90b1c10fb4R191-R192 to reuse them?